cli/src/main.rs

```diff
@@ -484,6 +487,23 @@ impl World for SystemWorld {
             .get_or_init(|| read(path).map(Buffer::from))
             .clone()
     }
+
+    fn today(&self, offset: Option<i64>) -> Option<Datetime> {
+        if self.current_date.get().is_none() {
```

Comment:
any particular reason why you used chrono in the CLI's today impl instead of something from the time crate? is it because chrono was already used in the CLI or does time not have what we want?

Reply:
Replaced it.